### PR TITLE
43 change currency unit

### PR DESCRIPTION
**Pull request by winnie-chaintope**

### CODE REVIEW

**Comment by winnie-chaintope** [comment]
> The description of BTC/satoshi unit still remains.
> 
> * tapyrusd.1
> * src/qt/README.md
> * test/functional/README.md
>   ...
> 
> Please check with grep.

@azuchi san, i have just committed the remaining changes

**Comment by azuchi** [comment]
@winnie-chaintope stil remains in 
* `src/qt/locale/bitcoin_xx.ts`
* `src/coincontroldialog.cpp`
* `src/test/minertests.cpp`

use grep.

**Comment by winnie-chaintope** [comment]
@azuchi, i have just committed my changes

File: src/qt/locale/bitcoin_af.ts [hunk 1]
```diff
@@ -62,12 +62,12 @@
         <translation>Ontvanger adresse</translation>
     </message>
     <message>
-        <source>These are your Bitcoin addresses for sending payments. Always check the amount and the receiving address before sending coins.</source>
-        <translation>Hierdie is die adresse vanwaar u Bitcoin betalings stuur. U moet altyd die bedrag en die adres van die ontvanger nagaan voordat u enige munte stuur.</translation>
+        <source>These are your TPC addresses for sending payments. Always check the amount and the receiving address before sending coins.</source>
```
Comment (azuchi) [inline src/qt/locale/bitcoin_af.ts]: `Bitcoin` should be replaced `Tapyrus`.

File: src/qt/locale/bitcoin_af.ts [hunk 2]
```diff
@@ -410,8 +410,8 @@
         <translation>Orebalk</translation>
     </message>
     <message>
-        <source>Request payments (generates QR codes and bitcoin: URIs)</source>
-        <translation>Versoek betalings (genereer QR-kodes en bitcoin: URI's)</translation>
+        <source>Request payments (generates QR codes and Tapyrus: URIs)</source>
```
Comment (azuchi) [inline src/qt/locale/bitcoin_af.ts]: URI scheme `bitcoin:` should be replaced `tapyrus:`.

File: src/qt/locale/bitcoin_da.ts [hunk 3]
```diff
@@ -434,8 +434,8 @@
         <translation>Faneværktøjslinje</translation>
     </message>
     <message>
-        <source>Request payments (generates QR codes and bitcoin: URIs)</source>
-        <translation>Anmod om betalinger (genererer QR-koder og “bitcoin:”-URI'er)</translation>
+        <source>Request payments (generates QR codes and tapyrus: URIs)</source>
+        <translation>Anmod om betalinger (genererer QR-koder og “Tapyrus:”-URI'er)</translation>
```
Comment (azuchi) [inline src/qt/locale/bitcoin_da.ts]: still remain wrong URI scheme `Tapyrus:`. it should be replaced `tapyrus:`

File: src/qt/locale/bitcoin_ca.ts [hunk 4]
```diff
@@ -1328,16 +1328,16 @@
...
+        <translation>No es pot iniciar Tapyrus: controlador click-to-pay</translation>
     </message>
     <message>
         <source>URI handling</source>
         <translation>Gestió d'URI</translation>
     </message>
     <message>
-        <source>'bitcoin://' is not a valid URI. Use 'bitcoin:' instead.</source>
-        <translation>'bitcoin://' no és una URI vàlida. Usi 'bitcoin:' en lloc seu.</translation>
+        <source>'Tapyrus://' is not a valid URI. Use 'Tapyrus:' instead.</source>
```
Comment (azuchi) [inline src/qt/locale/bitcoin_ca.ts]: still remain. please check all *.ts file.

File: src/qt/locale/bitcoin_ca.ts [hunk 5]
```diff
@@ -446,16 +446,16 @@
         <translation>Mostra la llista d'adreces de recepció i etiquetes utilitzades</translation>
     </message>
     <message>
-        <source>Open a bitcoin: URI or payment request</source>
-        <translation>Obre una bitcoin: sol·licitud d'URI o pagament</translation>
+        <source>Open a tapyrus: URI or payment request</source>
+        <translation>Obre una Tapyrus: sol·licitud d'URI o pagament</translation>
```
Comment (azuchi) [inline src/qt/locale/bitcoin_ca.ts]: still remain...

File: src/amount.h [hunk 6]
```diff
@@ -17,7 +17,7 @@ static const CAmount CENT = 1000000;
 /** No amount larger than this (in satoshi) is valid.
```
Comment (azuchi) [inline src/amount.h]: still remain satoshi.

Comment (winnie-chaintope) [reply src/amount.h]: ok
for 'bitcoin' in every README.md i will change to 'Tapyrus' also ?

**Pull request merged**